kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/message/store/AbstractJdbcMessageStoreImpl.java

```diff
@@ -103,8 +103,7 @@ protected long storeInternal(String topic, byte[] payload, int qos, boolean reta
 
             final long result;
 
-            try (PreparedStatement pstmt = c.prepareStatement(this.queries.getSqlStore(),
-                    new String[] { "id" })) {
+            try (PreparedStatement pstmt = c.prepareStatement(this.queries.getSqlStore(), new String[] { "id" })) {
```

Comment:
It looks like this file has only formatting related changes, if this is true can you please roll them back?